src/react/components/pages/connections/connectionForm.tsx

```diff
@@ -85,6 +86,12 @@ export default class ConnectionForm extends React.Component<IConnectionFormProps
                     </span>
                 </h3>
                 <div className="m-3 text-light">
+                    { isBrowser() && this.state.providerName === "bingImageSearch" &&
```

Comment:
is there a better way to compare for Bing Image? providerName (type) is only define in the register script. Is there another type comparison that can be done here? I don't want to use the display name.

Reply:
Nothing that I can think of right now - this is the first time we've wanted to add some additional text inline within the provider form.  Only other thing would be to define a "description" in the provider settings, but it would show for all host modes.